### PR TITLE
Master matts aboutus updates

### PR DESCRIPTION
**Pull request by mattdias**

### CODE REVIEW

File: app/views/forms/edit/document-selector.html [hunk 1]
```diff
@@ -50,37 +61,91 @@
...
+                                          <tr ng-repeat="doc in docs = (rawDocuments.docs | orderBy:sortType:sortReverse  | filter:keyword)" id="rec">
+                                            <td style="width:0;"><span class="bold">{{$index+1}}.</span></td>
+                                            <td  style="width:0;">  
+                                                <input ng-if="type==='checkbox'" type="checkbox" ng-model="doc.__checked" />
+                                                <input ng-if="type==='radio'" type="radio" name="{{question}}_selected" ng-model="doc.__checked" ng-value="true" />
+                                            </td>
+                                            <td class="type"> {{doc.schema_s | schemaShortName}}</td>
+                                            <td style="min-width:30%" class="title"> <a href="{{doc.url_ss}}" target="_new">{{doc.rec_title}}</a></td>
+                                            <td > {{doc.rec_countryName}}</td>
+                                            <td class="id color-blue"> <a href="{{doc.url_ss}}" target="_new">{{doc.uniqueIdentifier_s}}</a></td>
```
Comment (blaisefonseca) [inline app/views/forms/edit/document-selector.html]: use _blank

File: app/views/forms/view/view-articles.html [hunk 2]
```diff
@@ -13,37 +17,48 @@
...
+        <div class="row well bg-white">
+            <div  class="col-xs-12 col-sm-6 col-md-3 hidden-xs" ng-if="article.coverImage.url" >
+                <img class="thumbnail" style="max-width:300px;width:100%; width: 100%; margin-right:20px;" ng-if="article.coverImage.url" src="{{getSizedImage(article.coverImage.url, '400x400')}}">
             </div>
-            <div class="col-md-7">
-                <p class="news-date bg-white" >{{article.meta.createdOn | date}}</p>
-                <h1 style="font-size:260%;">{{article.title.en}}</h1>
-                <p ng-bind-html="article.content.en"></p>
+            <div class="col-xs-12" ng-class="{'col-sm-6 col-md-9' : article.coverImage.url}">
+                    <span class="news-date bg-white pull-right bold"  >{{article.meta.createdOn | date}}</span>
```
Comment (blaisefonseca) [inline app/views/forms/view/view-articles.html]: use the utcdate filter

File: app/views/forms/view/view-articles.html [hunk 3]
```diff
@@ -13,37 +17,48 @@
...
-                                        <span class="bg-red color-white news-date">{{article.meta.createdOn | date}}</span>
-                                </div>
-                                <div style="margin:5px;">
-                                    <h4>{{article.title.en}}</h4>
-                                    <div> <span ng-bind-html="(article.content.en | limitTo: 100)+(article.content.en.length > 100 ? '...' : '')" style="display:inline;"></span></div>
-                                </div>
-                            </div>
+                                <div  style="background-color:#f9f9f9;border-top:1px solid #ddd; border-bottom:1px solid #ddd;height:200px;margin-bottom:10px;">
+                                        <div  class="news-image" style="background-image: url({{::getSizedImage(article.coverImage.url, '400x400')}});" ng-class="{'sized':article.coverImage.url,'small-sized':!article.coverImage.url}"> 
+                                                <span class="bg-red color-white news-date">{{article.meta.createdOn | date}}</span>
```
Comment (blaisefonseca) [inline app/views/forms/view/view-articles.html]: `formatDateWithUtc`

File: app/views/forms/view/view-articles.js [hunk 4]
```diff
@@ -19,11 +19,23 @@ define(['app','underscore',
...
+        var breadcrumb = {    
+          label : 'Announcement',
+          originalPath : "articles",
+          param:'articles',
+          path:"/articles/"
+        }
+
       //---------------------------------------------------------------------
       function loadArticle(id){
           $q.when($http.get('https://api.cbd.int/api/v2017/articles/' + id))
```
Comment (blaisefonseca) [inline app/views/forms/view/view-articles.js]: should use articleService

File: app/services/articles-service.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+define(['app', 'underscore', './local-storage-service', './app-config-service',
```
Comment (blaisefonseca) [inline app/services/articles-service.js]: remove unused local-storage reference

File: app/services/articles-service.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+                                    }
+                                    value = replaceOembed(value);
+
+                                    return JSON.parse(value);
+                                }
+                        })
+                }
+                //================================================================================================================
+                function openInNewWindow(str) {
+                    return str.replace(/href=/g," target=\\\"_new_window_\\\" href=" );
```
Comment (blaisefonseca) [inline app/services/articles-service.js]: i think only _new is enough

Comment (mattdias) [reply app/services/articles-service.js]: ha!

**Comment by mattdias** [comment]
updated!

**Comment by mattdias** [comment]
Added some more changes.

**Pull request closed without merging**